misc/2017-09-20-ray-dillinger-if-id-known.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Ray Dillinger -  If I'd Known What We Were Starting (2017-09-20)
+translation_by: Julien Urraca
+categories: ['chaincode-curriculum']
```

Comment:
I don't think we need to introduce this category.

Totally acceptable to not have a category like https://btctranscripts.com/greg-maxwell/2020-11-05-greg-maxwell-yubikey-security/ or maybe introduce something related to posts/writing